queries.sql

```diff
@@ -7,3 +7,44 @@ SELECT pet_name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE PET_NAME = 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+/* SECOND PART */
+BEGIN;
+
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT SP1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO SAVEPOINT SP1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+COMMIT;
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/206017607-f0e760d7-3ec3-4199-9f28-b18c90d14542.PNG)

- [ ] You didn't add all the transactions in your queries.sql file which you can see in the image above ⬆️ Kindly add the missing transactions in your queries.sql file 😊
- [ ] Kindly take screenshots of your query results and add them in your pull request summary to follow the requirements 👍

Reply:
- [ ] as the previous CR already highlighted in the image you didn't create all the required transactions, please make sure to complete every step of the milestone before requesting for review.